<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGetDefaultDialog</class>
 <widget class="QDialog" name="PGetDefaultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../musredit.qrc">
    <normaloff>:/images/musrfit.xpm</normaloff>:/images/musrfit.xpm</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="fHelp_pushButton">
      <property name="text">
       <string>&amp;Help</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="fOk_pushButton">
      <property name="text">
       <string>&amp;OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fCancel_pushButton">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="fRunFileName_label">
      <property name="text">
       <string>Run File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fRunFileName_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="fBeamline_label">
      <property name="minimumSize">
       <size>
        <width>68</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Beamline</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fBeamline_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="fInstitute_label">
      <property name="minimumSize">
       <size>
        <width>68</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Institute</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fInstitute_comboBox">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>PSI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TRIUMF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPARC</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="fFileFormat_label">
      <property name="minimumSize">
       <size>
        <width>68</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>File Format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fFileFormat_comboBox">
      <item>
       <property name="text">
        <string>NeXuS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ROOT-NPP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ROOT-PPC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSI-BIN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MDU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MDU-ASCII</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MUD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WKM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASCII</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DB</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fOk_pushButton</tabstop>
  <tabstop>fCancel_pushButton</tabstop>
  <tabstop>fHelp_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../musredit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fCancel_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetDefaultDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fOk_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetDefaultDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fHelp_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PGetDefaultDialog</receiver>
   <slot>helpContent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>helpContent()</slot>
 </slots>
</ui>
